bats/katello-bats

```diff
@@ -34,8 +34,8 @@ for arg do
       swapfile)
         swapfile=true
         ;;
-      capsule)
-        katello_capsule=true
+      proxy)
+        katello_proxy=true
```

Comment:
Nitpick -- should we stick to `proxy_content` for consistency across all the re-names?